main.py

```diff
@@ -252,6 +252,21 @@ def list(self, dpid=None):
 
         return jsonify(switch_flows)
 
+    @listen_to('kytos.flow_manager.flows.(install|delete)')
+    def event_add_flow(self, event):
+        """Install or delete Flow on the switches through events.
+
+        Install or delete Flow from on switch identified by DPID.
+        """
+        try:
+            dpid = event.content['dpid']
+            flow_dict = event.content['flow_dict']
+            command = event.content['command']
+            switch = self.controller.get_switch_by_dpid(dpid)
+            self._install_flows(command, flow_dict, [switch])
+        except (KeyError, InvalidCommandError) as error:
+            log.error(f"Error to install flow: {error}")
```

Comment:
Break the try/except in two, one for each exception.

Reply:
Resolved in https://github.com/kytos/flow_manager/pull/129/commits/dfc4ba126992579fc11020a1291cfcea583f92bc